accounts/platform/rolesets.tf

```diff
@@ -76,6 +81,7 @@ module "super_dev_roleset" {
     local.identity_account_roles["ci_role_arn"],
     local.digitisation_account_roles["ci_role_arn"],
     local.experience_account_roles["ci_role_arn"],
+    local.dam_prototype_account_roles["ci_role_arn"],
```

Comment:
Does CI need access to this account? I don't think BuildKite will be doing much here.

Reply:
Same as above. It's ok not to use it.